runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
On Tue, Sep 29, 2015 at 05:19:20PM -0700, Alexander Morozov wrote:
> I said you my use-cases before - for registering application as
> working somewhere…

But “we just executed the code” doesn't necessarily map to “it's
working now” does it?  Most processes have some startup where they
read config files, open sockets, etc., etc., before they're ready to
start performing their primary function (e.g. serving web requests,
analyzing data files, etc., etc.).  So I agree that a pre-start hook
would be a poor place for “this application is ready to do real work”.
I just think that a post-start hook is still not the right place.  For
socket-based applications, “the process has created and bound to the
expected address” is probably the right place, but that's not when a
post-start hook would be firing.

> I wonder if you think that only your use-cases are true. Pls,
> believe mine real as well.

I'm sure your have a real use case, which is why I'm asking for more
details on what that use case is [1].  I'm less confident that a
post-start hook is the best solution to your problem, but that may be
just because I don't understand what that problem is :p.  Do you have
some code I can look at that demonstrates the problem you're trying to
solve with a post-start hook?

[1]: https://github.com/opencontainers/specs/pull/202#discussion_r40744864